mubench.reviewsite/src/Controllers/ReviewsController.php

```diff
@@ -28,10 +28,17 @@ public function getReview(Request $request, Response $response, array $args)
         $misuse_muid = $args['misuse_muid'];
 
         $experiment = Experiment::find($experiment_id);
-        $detector = Detector::find($detector_muid);
+        $detector = $this->getDetector($detector_muid);
         $ex2_review_size = $request->getQueryParam("ex2_review_size", $this->settings['default_ex2_review_size']);
 
-        $reviewer = array_key_exists('reviewer_name', $args) ? Reviewer::where(['name' => $args['reviewer_name']])->first() : $this->user;
+        $reviewer = $this->user;
```

Comment:
As for the detector, I would move this logic into a lookup method on the `Reviewer` class.